spec/integration/rails/persistence_spec.rb

```diff
@@ -491,10 +520,29 @@ def self.model_name
           allow(controller).to receive(:resource) { klass }
         end
 
-        it "does not require the FK to be a writable attribute" do
-          make_request
-          employee = Employee.last
-          expect(employee.positions.map(&:title)).to eq(["foo"])
+        context "and the overall request is create" do
+          subject(:make_request) { do_create(payload) }
+
+          it "does not require the FK to be a writable attribute" do
+            expect {
+              make_request
+            }.to change { Employee.count }.by(1)
+            employee = Employee.last
+            expect(employee.positions.map(&:title)).to eq(["foo"])
+          end
+        end
+
+        context "and the overall request is update" do
+          subject(:make_request) { do_update(payload) }
+
+          before do
+            payload[:data][:id] = employee.id.to_s
+          end
+
+          it "does not require the FK to be a writable attribute" do
```

Comment:
The title of the PR says "when associating on create" but this allows it on update too. Just checking that was intended.

Reply:
Nevermind, re-read the PR description and it makes sense.